### PR TITLE
Issue solving

### PR DESCRIPTION
**Pull request by DaluS**
This merge is just an ensemble of small improvement. 
It includes : 
* Allowing a better reading of the model already coded 
* Repairing a few plots 
* Reintroducing basinofattraction as an example of cockpit
* Removed the necessity of Func_Order in model

Nothing major, just a nicer base to exploit !

### CODE REVIEW

**Review by Didou09 — COMMENTED** [review]
Are you sur you are working from the devel branch ?

This issue of _core beign copied in itself does not exist in devel. 
This is the last lines of _core in devel:

image

Maybe you started from a different branch ?
In this case it is difficult to merge this PR because we might merge other changes without knowing too

File: pygemmes/_plots/_plot_timetraces.py [hunk 1]
```diff
@@ -152,7 +152,9 @@ def _plot_timetraces_check_dax(
             # set ylabels
             ylab = v0['symbol']
             if v0['units'] not in [None, '']:
-                ylab += f" (${v0['units']}$)"
+                unitemp = v0['units'].replace('$', '\$')
+                # print(unitemp)
+                ylab += f" ($ {unitemp} $)"
```
Comment (Didou09) [inline pygemmes/_plots/_plot_timetraces.py:157]: ok for me

**Review by Didou09 — COMMENTED** [review]
Same question as before, are you sure you started this from branch devel ?

On devel, this files is very small and looks like this:

image

File: pygemmes/_plots/_plot_timetraces.py (hunk 1 above)
Comment (DaluS) [reply pygemmes/_plots/_plot_timetraces.py:157]: > Same question as before, are you sure you started this from branch devel ?
> 
> On devel, this `pygemmes/__init__.py` files is very small and looks like this:
> 
> image

Yes ! I did it on purpose as a few general functions that are showing/using pygemmes in general

**Comment by DaluS** [comment]
> Are you sur you are working from the devel branch ?
> 
> This issue of _core beign copied in itself does not exist in devel. This is the last lines of _core in devel:
> 
> image
> 
> Maybe you started from a different branch ? In this case it is difficult to merge this PR because we might merge other changes without knowing too

I checked in the log, I once copied the file in itself, then once I saw it removed it. I removed the non-modified code, so it's a branch from devel and compatible with the previous version

**Review by Didou09 — COMMENTED** [review]
Commit 1:   #d9df2d576bc71c5
---------------

* `__init__.py`: ok, presets re-included in `get_summary()` (was removed by previous commits)

I am ok with this PR, the 2 caveats from PR #116 are remaining: 
* data for non-regression tests needed
* `_core.py`: a part of `Hub` methods have been copy-pasted in itself (lines 448 - 1251)

**Review by Didou09 — COMMENTED** [review]
Commit 2 :    #696632ea6f83
--------------

* `__init__.py`: ok
    * removed func_order 
    * added ode and statevar in get_available_models(), removed functions nb. 

* `_model_GK-Reduced.py`: ok, removed func_order (maybe also remove from other models ?)

I am ok with this commit, the previous 2 caveats remain

**Review by Didou09 — COMMENTED** [review]
Commit 3:      #af3e20eccfaa5169
--------------

* Overlooked files in `Examples/` and `NotIncluded/`

I am ok with this commit, the previous 2 caveats remain

**Review by Didou09 — COMMENTED** [review]
Commit 4:    #57de46305c1c7
--------------

* `_plots/_plot_timetraces.py`: ok, minor change to str formatting

I am ok with this commit, the previous 2 caveats remain

**Review by Didou09 — COMMENTED** [review]
Commit 5:      #6b96264669ed
--------------

* `__init__.py`: ok, minor changes to routines

* `_plots/_plot_timetraces.py`: ok, minor str reformatting

I am ok with this commit, the previous 2 caveats remain

**Review by Didou09 — CHANGES_REQUESTED** [review]
Commit 6:         #0e3767a2a2b5f6
--------------

* Overlooking files in `Examples/`

* `_core.py`: not ok, removed the wring line interval (794-1594 instead of 448-1251)

* `_plots/_plots.py`: ok, minor change to select system

I am not ok with this commit, 2 caveats remains:
* data for non-regression tests needed
* the wrong line interval was removed in `_core.py` (794-1594 instead of 448-1251)

**Comment by Didou09** [comment]
I will fix the last caveat and merge this PR into branch ExtensiveModels, which will then allow to fallback to PR #116

**Review by Didou09 — APPROVED** [review]
Done

**Pull request merged**